### PR TITLE
Switch to Suru cursor theme

### PR DESCRIPTION
**Pull request by merlijn-sebrechts**
This is the second part of https://github.com/ubuntu/communitheme-snap-helpers/pull/13

### CODE REVIEW

**Comment by didrocks** [comment]
A new test snap version is available using: `snap refresh communitheme --channel=edge/communitheme-snap-helpers-pr15`.
Further available updates will track that pull request then.

Switch back to stable or edge snap with `snap refresh communitheme --stable` or `snap refresh communitheme --edge` once you are done with it!
You may want to download as well `gtk-common-themes` snap from `edge/communitheme-snap-helpers-pr15` channel to test your snaps with those changes.

Replace `refresh` with `install` if you haven't installed the snap yet.

**Comment by didrocks** [comment]
A new test snap version is available using: `snap refresh communitheme --channel=edge/communitheme-snap-helpers-pr15`.
Further available updates will track that pull request then.

Switch back to stable or edge snap with `snap refresh communitheme --stable` or `snap refresh communitheme --edge` once you are done with it!
You may want to download as well `gtk-common-themes` snap from `edge/communitheme-snap-helpers-pr15` channel to test your snaps with those changes.

Replace `refresh` with `install` if you haven't installed the snap yet.

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -33,13 +33,13 @@ parts:
...
     plugin: meson
     meson-parameters: ["--prefix=/"]
-  cursor:
-    source: https://github.com/ubuntu/cursor-communitheme.git
-    plugin: dump
-    organize:
-      Communitheme_13: share/icons/communitheme
-    stage:
-      - share/icons/
+    override-build: |
```
Comment (didrocks) [inline snap/snapcraft.yaml:36]: I would keep the normal `build` step, and rather do things in the `install` step. You can take some inspirations from https://github.com/snapcrafters/gtk-common-themes/blob/master/snap/snapcraft.yaml#L178

**Review by didrocks — CHANGES_REQUESTED** [review]
Thanks for proposing this! Check my comments, I think it's worth doing this, which is a bit better IMHO than restarting snapcraft inside snapcraft ;)

**Comment by merlijn-sebrechts** [comment]
Using the `install` step gives a deprecation warning which suggests this method: https://docs.snapcraft.io/deprecation-notices/dn9

I personally also prefer `override-build`  because it's more clear to me what is actually happening and when. Less "magic".

**Comment by didrocks** [comment]
Interesting… Ok, not sure I'm a really big fan of this, but ok, as it's in the doc :p

**Review by didrocks — APPROVED** [review]

**Pull request merged**